packages/compose/README.md

```diff
@@ -288,11 +288,13 @@ _Returns_
 In some circumstances, such as block previews, all focusable DOM elements
 (input fields, links, buttons, etc.) need to be disabled. This hook adds the
 behavior to disable nested DOM elements to the returned ref.
+If you can, prefer the use of the inert HTML attribute.
```

Comment:
We should use formatting to make it a bit clear that `inert` is the name of an HTML attribute, rather than merely an adjective. Also, I think there should be another newline here to create a paragraph break.

```suggestion

If you can, prefer the use of the `inert` HTML attribute.
```